docs/docs/metrics.md

```diff
@@ -0,0 +1,150 @@
+ number of currently processing status updates.
+ 
+ <span class="label label-default">v0.15</span>
+ `service.mesosphere.marathon.core.task.update.impl.TaskStatusUpdateProcessorImpl.publishFuture` (timer) -
+ times processing status updates.
+
+### Configuration update processing
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.state.GroupManager.queued` (gauge) - number of queued app configuration
+updates. The maximum is configured with `--max_queued_root_group_updates`.
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.state.GroupManager.processing` (gauge) - number of currently processed app configuration
+updates. Since we serialize these updates, this is either 0 or 1.
+
+### Repositories
+
+Marathon stores its permanent state in "Repositories". These are the most important ones:
+
+* `GroupRepository` for app configurations and groups.
+* `TaskRepository` for the latest known task state. This is the repository with the largest
+  data churn.
+
+There are also the
+
+* AppRepository for versioned app configuration.
+* DeploymentRepository for currently running deployments.
+* TaskFailureRepository for the last failure for every application.
+* and more...
+
+For each of the repositories we have statistics about read and write requests. Substitute `*` with
+the repository of interest:
+ 
+`service.mesosphere.marathon.state.*.read-request-time.count` - the number of read requests.
+
+`service.mesosphere.marathon.state.*.read-request-time.mean` - the exponential weighted average of
+the read request times.
+
+`service.mesosphere.marathon.state.*.write-request-time.count` - the number of write requests.
+
+`service.mesosphere.marathon.state.*.write-request-time.mean` - the exponential weighted average of
+the write request times.
+
+Please note that prior to <span class="label label-default">v0.15</span> these were not measured correctly.
+
+### Requests
+
+`org.eclipse.jetty.servlet.ServletContextHandler.dispatches` (timer) - The
+number of HTTP requests received by Marathon is available under `.count`.
+There are more metrics around HTTP requests under the
+`org.eclipse.jetty.servlet.ServletContextHandler` prefix.
+For more infos, look at
+[the code](https://github.com/dropwizard/metrics/blob/796663609f310888240cc8afb58f75396f8391d2/metrics-jetty9/src/main/java/io/dropwizard/metrics/jetty9/InstrumentedHandler.java#L41-L42).
+
+### JVM
+
+`jvm.threads.count` (meter) - the total number of threads. If this is above >500, this
+is generally a bad sign.
+
+`jvm.memory.total.used` (meter) - the total number of bytes used by the Marathon JVM.
+
+## Instrumented method calls
+
+These metrics are created automatically by instrumenting certain classes in our code base which can be disabled
+by `--disable_metrics` which will not disable all metrics. Generally, these timers can be very valuable in diagnosing
```

Comment:
I'm very bad with prepositions in english. But I think the following sounds better: s/by `--disable_metrics`/with `disable_metrics`